### ISSUE TITLE
Travis build broken on master 

### ISSUE THREAD

**Issue by joka** [liqd/adhocracy3#2052]
For some time all travis tests fails because the build is broken. 
Is someone working on this?

For example here: https://s3.amazonaws.com/archive.travis-ci.org/jobs/108242113/log.txt

**Comment by xi** [comment]
There were some tests failing for sometime, and we are working on it whenever we have time. However, the referenced output looks like there is an issues with bower. Could it be network issues?

**Comment by joka** [comment]
> There were some tests failing for sometime, and we are working on it whenever we have time. However, the referenced output looks like there is an issues with bower. Could it be network issues?

bower can resolve all  urls: `bower [36mresolved     [39m git://github.com/lodash/lodash.git#3.10.1`
This looks similar to the builtout problem with the bower part we have from time to time.

**Comment by xi** [comment]
I am not so sure: For me the last line of bower output looks like this:

```
bower progress      DefinitelyTyped#54f064352a3615443f7bff4198078db15e28247b Resolving deltas:  73% (25230/34537)
```

It seems like it is not really done.

**Comment by xi** [comment]
This might get fixed by #2141, but that is only a guess.

**Comment by xi** [comment]
Hasn't happened for a while. Closing.
